server/server.go

```diff
@@ -143,16 +180,18 @@ func (s *Server) HandlePush(ctx context.Context, e *PushEvent) error {
 	}
 	comments := s.concurrentRequest(ctx, logger, conf, send)
 
-	s.post(ctx, logger, e, comments)
-	s.status(ctx, logger, e, SuccessAnalysisStatus)
+	if err := s.post(ctx, logger, e, comments); err != nil {
+		return fmt.Errorf("posting analysis failed: %s", err)
```

Comment:
Same comment as in HandleReview: I think this will leave the github status in pending, and we should set it to lookout.ErrorAnalysisStatus.